app/scripts/components/data-catalog/container.tsx

```diff
@@ -17,16 +17,10 @@ import { FeaturedDatasets } from '$components/common/featured-slider-section';
 export default function DataCatalogContainer() {
   return (
     <PageMainContent>
-      <LayoutProps
-        title='Data Catalog'
-        description={getString('dataCatalogBanner').other}
-      />
-      <PageHero
-        title='Data Catalog'
-        description={getString('dataCatalogBanner').other}
-      />
+      <LayoutProps title='Data Catalog' description={getString('dataCatalogBanner').other} />
```

Comment:
Nit: Should we make the titles here more dynamic (ie. to arrive as props from the DataCatalogContainer?)

Reply:
Ah this is a great point - we would need this as a prop once we start exporting layout components.

Reply:
Discussed! This is layout related which will eventually be dictated at the page level, the page level can decide which layout component they would use and the title to pass in, this Layout Component already takes in title as a prop as well and title passed in could be made to be dynamic on the instance side. Given that this container is temporary to just support current instances, this shouldn't matter